.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
why is this empty file required?

Reply:
There was a PR beside mine that I had to merge with mine and fix some conflicts, so maybe that came with it, but I honestly don't remember ever making this file or its use. i can delete it if need be.